source/slang/slang-ir-dll-import.cpp

```diff
@@ -0,0 +1,249 @@
+// slang-ir-dll-import.cpp
+#include "slang-ir-dll-import.h"
```

Comment:
Minor point, but we probably need to think about naming for this attribute and associated functionality over time. `DllImport` is very Windows-specific terminology (even in the C# feature works on other platforms).